.github/workflows/volume_group_operator_ci.yaml

```diff
@@ -0,0 +1,98 @@
+
+name: volume-group-operator-ci
+on:
+  push:
+    branches:
+      - develop
+      - main
+  pull_request:
+    types: 
+      - opened
+      - reopened
```

Comment:
Can you open a ticket to support the PR merge?

Reply:
I changed it so that it is trigged for every PR event